docs/s3.md

```diff
@@ -0,0 +1,153 @@
+# S3
+
+Action uses Amazon S3 for three purposes:
+
+   1. Serving manually managed assets, such as logos
+   2. Serving builds bundled with webpack
+   3. Storing and serving user-generated assets
+
+Action uses a single S3 bucket. Within the bucket, the directory structure
+is as follows:
+
+```
+   S3 Bucket
+   =========
+   |
+   |--- static/                     1) Manually managed assets
+   |
+   |--- :instance/
+      |--- build/                   2) Builds bundled with webpack
+      |  |--- :vX.Y.Z/
+      |
+      |--- store/                   3) User-generated assets
```

Comment:
that should be OK, as long as mirai doesn't decide that they're really interested in a client's assets...